pkg/drivers/kic/oci/oci.go

```diff
@@ -64,6 +63,23 @@ func CreateContainerNode(p CreateParams) error {
 		"--label", fmt.Sprintf("%s=%s", nodeRoleKey, p.Role),
 	}
 
+	// volume path in minikube home folder to mount to /var
+	hostVarVolPath := filepath.Join(localpath.MiniPath(), "machines", p.Name, "var")
+	if err := os.MkdirAll(hostVarVolPath, 0777); err != nil {
+		return errors.Wrapf(err, "create var dir %s", hostVarVolPath)
+	}
+
+	if p.OCIBinary == Podman { // enable execing in /var
+		// podman mounts var/lib with no-exec by default  https://github.com/containers/libpod/issues/5103
+		runArgs = append(runArgs, "--volume", fmt.Sprintf("%s:/var:exec", hostVarVolPath))
+	}
+	if p.OCIBinary == Docker {
+		runArgs = append(runArgs, "--volume", fmt.Sprintf("%s:/var", hostVarVolPath))
```

Comment:
Could we limit this so that it is only mounted this way on Linux? It potentially offers poor performance when a VM is involved.